api/admin.js

```diff
@@ -20,6 +22,57 @@ router.put("/:email", async (req, res) => {
   return res.json({ hackerProfile: newHackerProfile });
 });
 
+router.get("/profiles", async (req, res) => {
+  const Op = sequelize.Op
+  const { query } = req.query;
+  const flexQuery = '%' + query + '%'
+  try {
+    const profiles = await models.HackerProfile.findAll({
+      where: {
+        [Op.or]: [
+          {
+            email: {
+              [Op.like]: flexQuery
+            }
+          },
+          {
+            firstName: {
+              [Op.like]: flexQuery
+            }
+          },
+          {
+            lastName: {
+              [Op.like]: flexQuery
+            }
+          },
+        ]
+      }
+    });
+    return res.json({
+      profiles
+    });
+  } catch (e) {
+    return res.status(500).json({ error: e.message });
```

Comment:
Set this status to 400, if you're expecting user facing failure

Reply:
Not really expecting user failure, probably more important if server crashes during event lol